effectly/src/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,86 @@
+import { CalendarWrapper, EventsRows, NoEventsInfo } from "./Calendar.styles";
+
+import DatePicker from "./DatePicker/DatePicker";
+import Event from "./Event/Event";
+
+import useCalendarData from "../../hooks/Calendar/useCalendarData";
+import useDatePick from "../../hooks/Calendar/useDatePick";
+
+import { convertDate, filterEvents, getEventsMath } from "./helpers/events";
+import getRandomColor from "./helpers/getRandomColor";
+
+type CalendarProps = {
+  className: string;
+  userId: string;
+  children?: React.ReactNode;
+};
+
+const Calendar = ({ className, userId }: CalendarProps) => {
+  const { calData: calendarData } = useCalendarData(userId);
+  const {
+    calendarDay: visibleDay,
+    weekDayString: weekDay,
+    incrementDay,
+    decrementDay,
+  } = useDatePick();
+
+  const { sortedAndFilteredEvents: events } = filterEvents(
+    calendarData,
+    visibleDay
+  );
+
+  const eventsFetched = events.length !== 0;
+
+  return (
+    <CalendarWrapper className={className}>
+      {!calendarData ? (
+        "Loading..."
```

Comment:
It might be used in other places. Maybe better would be to move that to file with consts variables ?

Reply:
Done with [c4327f7](https://github.com/wooczak/effectly/pull/8/commits/c4327f7a9b13ae67004aceb8e349558aee8cef2e)